giotto/utils/_docs.py

```diff
@@ -0,0 +1,134 @@
+"""Utilities for docstring building."""
+# License : Apache 2.0
```

Comment:
If we really need to include the GPL license on every file, then we should probably include it here too

Reply:
Well spotted, I missed it in my refactoring due to the space before the colon!

Reply:
Done.